nightswatcher/nightswatcher.py

```diff
@@ -135,6 +135,7 @@ def is_stable(commit_number):
                         'build_cervantes',
                         'build_kindle', 'build_legacy_kindle',
                         'build_kindle5', 'build_kindlepw2',
+                        'build_kindlehf'
```

Comment:
Oof, it just took me way too long to find this typo. :-D

```
frozenset({'build_kindlehfbuild_kobo', 'build_sony_prstux', 'build_kindle5', 'build_remarkable', 'build_legacy_kindle', 'build_kindlepw2', 'build_pocketbook', 'build_cervantes', 'build_kindle'})
```

Reply:
lol :P